open-grant-proposals/open-proposal-blockchain-package-manager.md

```diff
@@ -0,0 +1,183 @@
+# Open Grant Proposal: `Blockchain Package Manager - FileCoin Package`
+
+**Name of Project: FileCoin BPM package**
+
+**Proposal Category:** `devtools-libraries`
+
+**Proposer:** `JonasPf`
+
+**Do you agree to open source all work you do on behalf of this RFP and dual-license under MIT and APACHE2 licenses?:** "Yes"
+
+# Project Description
+
+Create a basic bootstrap application using the Blockchain Package Manager that will install the FileCoin Full Node, all Blockdaemon dependencies, and anything else required to run, manage and monitor a FileCoin node. 
+
+In addition, we will add FileCoin to our principal node-marketplace used by professionals and enterprises to evaluate which protocol to build on.
+
+BPM supports a plugin/package architecture for different Blockchain protocols. FileCoin support will be implemented as a plugin/package. This allows to configure and launch FileCoin nodes on any supported cloud or BYOI infrastructure in a standardized way. The underlying OS can be Linux or OSX (needs Docker). BPM offers an auto-upgrade mechanism for plugins/packages to ensure users always run the latest versions.
+
+Blockchain node events, logs, telemetry, health data will be forwarded to the Blockdaemon data pipeline for storage and analysis. Data will be analyzed and displayed to Blockdaemon users and optionally be made available to paying customers. The Blockdaemon support infrastructure will automatically detect anomalies and alert customers if that happens.
+
+The goal is to run bpm to install, download, and configure a node that connects to the FileCoin testnet as well as the mainnet when it launches.
+
+## Value
+
+### What are the benefits to getting this right?
+
+Standardized way of launching FileCoin nodes making it easier for users to configure and monitor and maintain a FileCoin node. This will improve the adoption rate of FileCoin. Monitoring will provide an audit trail for SLAs. Each BPM package comes with a test suite to test the correct functionality of a node.
+
+### What are the risks if you don't get it right?
+
+Running FileCoin nodes will be restricted to users who can afford the engineering resources to do so. The adoption rate of FileCoin will suffer. Monitoring the overall node health and performance will remain difficult.
+
+### What are the risks that will make executing on this project difficult?
+
+* Logs are not in a format that we can consume
+* Commands cannot be easily added to the plugin for technical reasons
```

Comment:
Is this a potential risk or something you have already seen through working with Filecoin?